subprojects/plugins/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -275,14 +275,14 @@ public void generate() {
         generator.generateWindowsScript(getWindowsScript());
     }
 
-    @Internal
-    private Iterable<String> getRelativeClasspath() {
-        return Iterables.transform(getClasspath().getFiles(), new Function<File, String>() {
+    @Input
+    private List<String> getRelativeClasspath() {
+        return Lists.newArrayList(Iterables.transform(getClasspath().getFiles(), new Function<File, String>() {
```

Comment:
Actually, it returns a `Set` since `getClasspath().getFiles()` is a `Set`. I reverted back to returning an `Iterable` and using `CollectionUtils`.